test/Feature/HLSLLib/sign.32.test

```diff
@@ -176,6 +176,9 @@ DescriptorSets:
 # https://github.com/microsoft/DirectXShaderCompiler/issues/7512
 # XFAIL: DXC-Vulkan
 
+# No idea what is going on here, but Out1 and Out3 are coming through as 0's.
+# XFAIL: Clang && Vulkan-Darwin
```

Comment:
Particle life is probably harder, but this one feel like we should have a follow up task to figure out what is going wrong.